.ci/ci.yml.tmpl

```diff
@@ -388,6 +388,15 @@ jobs:
               get_params:
                   skip_clone: true
 
+          # TODO(emilymye): Clean up create-or-update-pr script and aggregate
+          # this task into that one
+          # (since we should do it while updating downstreams)
+          - task: downstream-changelog-metadata
+            file: magic-modules/.ci/magic-modules/downstream-changelog-metadata.yml
```

Comment:
Could you name this to `terraform-downstream-changelog-metadata`?

Reply:
^ why I didn't do it - on the other hand, if you want to name this by repo because every other job is fairly repo specific and it could be misleading, I'm not against it (we can always rename)